Packs/RetarusSecureEmailGateway/ModelingRules/RetarusSecureEmailGateway/RetarusSecureEmailGateway.xif

```diff
@@ -0,0 +1,144 @@
+[RULE: retarus_secure_email_gw_common_fields]
+alter
+        tmp_description = if(direction = "INBOUND", "Inbound Email", "Outbound Email"),
+        source_ipv4 = arrayindex(regextract(sourceIp , "(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})"),0),
+        source_ipv6 = arrayindex(regextract(sourceIp, "([a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5})"),0)
+| alter
+        tmp_description_2 = if(
+        status = "ACCEPTED", ", Email Accepted", 
+        status = "REJECTED", ", Email Rejected",
+        status = "INFECTED" ,", Email Is Infected", 
+        status = "DETECTED", ", Malicious File/ URL Detected", 
+        status = "SUSPICIOUS",", Suspicious File/ URL Found" )
+| alter
+        xdm.email.origination_timestamp = parse_timestamp("%F %X %z",to_string(ts)),
+        xdm.observer.version = version,
+        xdm.observer.name = host, 
+        xdm.event.original_event_type = type,
+        xdm.source.ipv4 = source_ipv4,
+        xdm.source.ipv6 = source_ipv6,
+        xdm.email.recipients = arraycreate(recipient),
+        xdm.session_context_id = rmxId,
+        xdm.observer.action = status,
+        xdm.event.description = concat(tmp_description , tmp_description_2 ),
+        xdm.event.type = class,
+        xdm.email.message_id = mimeId;
+
+
+[RULE: pzd_fields]
+alter 
+        extract_threatType = metaData -> threatType,
+        extract_url = metaData -> uniformResourceLocator,
+        extract_hash = metaData -> hashFunction,
+        extract_checksum = metaData -> checksum,
+        extract_mimeType = metaData -> mimeType,
+        extract_details = metaData -> details
+
+| alter
+        xdm.email.mime = extract_mimeType,
```

Comment:
After doing some reading, lets keep only xdm.email.mime and remove xdm.email.attachment.file_type please